charts/hedera-block-node/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+appVersion: "0.1.0-SNAPSHOT"
+description: A Helm chart for Hedera Block Node
+home: https://github.com/hashgraph/hedera-block-node
+keywords:
+  - blocknode
+  - dlt
+  - hedera
+  - hashgraph
+maintainers:
+  - name: Hedera Block Node Team
+    email: blocknode@hashgraph.com
```

Comment:
We don't have a dedicated email for the blocknode right now.
Should we create one?

Reply:
We can figure this out.